updating selected destination path for clip, & refresh for all folders

jackgrossman18 opened the pull request:
ref #1639

Commit:
updating selected destination path for clip, & refresh for all folders

brianhatchl commented on modules/Hoot/tools/clipDataset.js:
```diff
@@ -193,7 +193,7 @@ export default class ClipDataset {
                 datasetName = row.select( '.datasetName' ),
                 outputName  = row.select( '.outputName' ),
-                folderId    = row.select( '.outputPath').property( '_value' );
+                folderId    = Hoot.folders.findBy( 'path', row.select( '.outputPath' ).property( 'value' ) ).id;
```
The `_value` property should have the folder id.  Was this change correcting something else?

jackgrossman18 replied:
Before this change, you couldn't change the destination path correctly. 

If the folder path provided when the modal became available would be the folder path the clipped dataset was sent to, regardless of what you had selected the path to be.

brianhatchl replied:
Looking elsewhere in the code, I think it should be `.attr('_value')`
that might fix it.

Commit:
updating setting the folder path for clipDataset

brianhatchl reviewed (changes requested):
There is one more to be replaced on line 136.

Commit:
requested change

brianhatchl committed:
Don't need separate refreshes if calling refreshAll

in the future we might think about moving the event emit
inside refreshAll, it seems like it's always called next
